[doc] Add kwokctl metrics (#452)
kubernetes-sigs/kwok: site/content/en/docs/user/kwokctl-metrics.md

# `kwokctl` Metrics

{{< hint "info" >}}

This document walks you through how to enable Metric on a `kwokctl` cluster

{{< /hint >}}

> ## Create a cluster with prometheus

Reviewer: ```suggestion
## Create a cluster with Prometheus
```

``` bash
kwokctl create cluster --prometheus-port 9090
```

> ## Create grafana dashboard with prometheus datasource

Reviewer: ```suggestion
## Create Grafana dashboard with Prometheus data source
```

``` bash
docker run -d --name=grafana -p 3000:3000 docker.io/grafana/grafana:9.4.7
```

1. Open your web browser and go to [http://localhost:3000]
2. On the login page, enter `admin` for username and password

> 3. Add Prometheus Datasource `http://host.docker.internal:9090` on Grafana

Reviewer: ```suggestion
3. Add the Prometheus data source, `http://host.docker.internal:9090`, on Grafana
```

4. Import via [grafana.com code] `16248` on Grafana

> Now you can see the grafana dashboard for the cluster

Reviewer: ```suggestion
Now you can see the Grafana dashboard for the cluster.
```
